src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsHandler.cs

```diff
@@ -27,18 +27,22 @@ public void Handle(List<PostProcessor> postProcessors, Manifest manifest, string
                 throw new ArgumentNullException(nameof(outputFolder));
             }
 
-            foreach (var postProcessor in postProcessors)
+            using (new LoggerPhaseScope("HandlePostProcessors", LogLevel.Verbose))
             {
-                using (new LoggerPhaseScope($"Process in post processor {postProcessor.ContractName}", LogLevel.Verbose))
+                foreach (var postProcessor in postProcessors)
                 {
-                    manifest = postProcessor.Processor.Process(manifest, outputFolder);
-                    if (manifest == null)
+                    using (new LoggerPhaseScope($"Processing {postProcessor.ContractName}", LogLevel.Verbose))
```

Comment:
new added logger phase prefix "HandlePostProcessors", so I think "processing" is enough?

---
In reply to: [96337779](https://github.com/dotnet/docfx/pull/1186#discussion_r96337779) [](ancestors = 96337779,96337376)